cypress/e2e/position.test.ts

```diff
@@ -0,0 +1,11 @@
+describe('Position', () => {
+  it('shows an valid state on a supported network', () => {
+    cy.visit('/pools/1')
+    cy.contains('UNI / ETH')
+  })
+
+  it('shows an invalid state on a supported network', () => {
+    cy.visit('/pools/788893')
+    cy.contains('To view a position, you must be connected to the network it belongs to.')
```

Comment:
I expected to see an assertion here instead of just a dom query. The failure case for `contains` is continuous retries until the test times out, so I think you should add a dedicated timeout and an explicit standalone assertion here.

Reply:
i based on https://docs.cypress.io/guides/references/best-practices#Text-Content